arches/app/templates/views/components/widgets/edtf.htm

```diff
@@ -10,67 +10,71 @@
         <div class="form-group">
           <div class="edtf-style-tools-collapser" data-bind=" click: function() {
               showEDTFFormats(!showEDTFFormats());
-          }">
-              <i class="fa" data-bind="{
+          }" tabindex="0" onkeyup="if(event.which == 13 || event.keyCode == 13){ $(this).trigger('click'); }">
+            <i class="fa" data-bind="{
                   css: {
                       'fa-caret-down' : !showEDTFFormats(),
                       'fa-caret-up' : showEDTFFormats(),
                   }
               }"></i>
-              <span data-bind="text: showEDTFFormats() ? '{% trans "EDTF Formats" %}' : '{% trans "EDTF Formats" %}'"></span>
+            <span data-bind="text: showEDTFFormats() ? '{% trans " EDTF Formats" %}' : '{% trans "EDTF Formats" %}'"></span>
           </div>
-          <div class="form-horizontal" data-bind="if: showEDTFFormats">
-              <div class="edtf-style-tools-panel">
-                  <div class="widget-help-panel">
-                    <h2 class="">{% trans "Extended Date/Time Formats (EDTF)" %}</h2>
-                    <h3 class="">
-                      {% trans "The EDTF datatype allows you to describe dates (even uncertain dates).  You can find a summary of the standard here:" %}
-                      <a class="" target="_blank" data-bind="attr: { href: 'https://www.loc.gov/standards/datetime'}, click: function() { return true;}, clickBubble: false">{% trans "EDTF Date Specfication (Library of Congress)" %}</a>
-                    </h3>
-                    <h3 class="">
-                      {% trans "Some common encodings:" %}
-                    </h3>
+          <div class=" form-horizontal" data-bind="if: showEDTFFormats">
+                <div class="edtf-style-tools-panel" tabindex="0">
+                    <div class="widget-help-panel">
+                        <h2>{% trans "Extended Date/Time Formats (EDTF)" %}</h2>
+                        <h3>
+                            {% trans "The EDTF datatype allows you to describe dates (even uncertain dates). You can
```

Comment:
the `trans` tags here and below on lines 30-31 are problematic due to the line feeds contained therein -- in the UI the `trans` tags are visible because they are not being rendered on the backend.

Reply:
Now fixed in latest commit